detect/errorloaderdetector.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+
+from .detector import Detector
+
+
+class ErrorsLoaderDetector(Detector):
+    """
+    Detector that loads a table of constant errors from csv.
+    """
+    def __init__(self, name, fpath):
```

Comment:
Add a default value for the `name` parameter, and move it last, similar to `NullDetector`

Reply:
Done.

Note: setting a default value for name might encourage using the same detector name for different fpaths (detectors), and since this detector loads data at __init__ (not at setup or detect_noisy_cells).